vizro-core/src/vizro/models/types.py

```diff
@@ -105,14 +106,61 @@ def validate_captured_callable(cls, value, info: ValidationInfo):
     if isinstance(value, (export_data, filter_interaction)):
         return value
 
+    try:
+        # Downside of validation is that it gets raised multiple times if there are multiple graphs in the dashboard.
```

Comment:
Why do we need to validate `callable_defs` at all? What happens if you don't do this? Why/how/when could someone specify it incorrectly?